src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -826,9 +826,9 @@ private void TouchEverything(Solution solution)
             {
                 foreach (var document in project.Documents)
                 {
-                    document.GetTextAsync().PumpingWait();
-                    document.GetSyntaxRootAsync().PumpingWait();
-                    document.GetSemanticModelAsync().PumpingWait();
+                    document.GetTextAsync().Wait();
+                    document.GetSyntaxRootAsync().Wait();
+                    document.GetSemanticModelAsync().Wait();
```

Comment:
can't remember exactly why it used PumpingWait but by looking at the code, test uses things like OnDocumentAdded and etc which are supposed to be used in UI thread (and workspace events happen on UI thread). so using just Wait might have caused deadlock before.